addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -183,6 +184,15 @@ const LinkPopoverWidget = Widget.extend({
         $('#wrapwrap').data('wysiwyg').odooEditor.execCommand('unlink');
         ev.stopImmediatePropagation();
     },
+    /**
+     * Avoids intra-popup links selection when panel is clicked.
+     *
+     * @private
+     * @param {Event} ev
+     */
+    _onPanelClick(ev) {
+        ev.stopImmediatePropagation();
+    },
```

Comment:
Why not I guess :+1: But scary the editor still allows to edit stuff that is not explicitly declared as editable (or is the popover actually put inside an editable content? :scream:)

Reply:
It actually did not fully work: clicking on external parts of the popup was not caught (e.g. the bubble arrow)
I added a `user-select: none;` on the popup CSS class instead.